### ISSUE TITLE
LaTeX beautifier suggestion

### ISSUE THREAD

**Issue by rudvfaden** [LaTeXing/LaTeXing#145]
It would be cool if you could implement [this](https://github.com/cmhughes/latexindent.pl) LaTeX auto code formatting script.

**Comment by csch0** [comment]
Great idea, but not sure how useful this will be... have your tried this on more complex LaTeX files??

**Comment by rudvfaden** [comment]
Yes. Think it works pretty great. But of course it may not be to
everybody's liking

On Wed, May 20, 2015 at 10:46 AM, csch0 <notifications@github.com> wrote:

> Great idea, but not sure how useful this will be... have your tried this
> on more complex LaTeX files??
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LaTeXing/LaTeXing/issues/145#issuecomment-103813014>.
>



-- 
Rud

**Comment by csch0** [comment]
Ok.. I never tried it so far... since perl is already installed I will give it a go and add this temporary to one of the next testing versions and see how it works.
